config/config.go

```diff
@@ -1045,7 +1044,6 @@ func SetupViper(v *viper.Viper, filename string) {
 	v.SetDefault("gdpr.tcf2.special_purpose1.enabled", true)
 	v.SetDefault("gdpr.tcf2.purpose_one_treatement.enabled", true)
 	v.SetDefault("gdpr.tcf2.purpose_one_treatement.access_allowed", true)
-	v.SetDefault("gdpr.amp_exception", false)
```

Comment:
What do you think of providing a warning to the host if this is enabled to guide them to understanding the change?